### PR TITLE
Button logic is updated as advised by Agam by following React way

### PR DESCRIPTION
**Pull request by itoogii**
Updated the play and record buttons

### CODE REVIEW

File: src/pages/Popup/Popup.jsx [hunk 1]
```diff
@@ -1,62 +1,67 @@
...
 import Greetings from '../../containers/Greetings/Greetings';
 import './Popup.css';
 import '../../assets/fontawesome/css/fontawesome.min.css';
 import '../../assets/fontawesome/css/solid.min.css';
 
 const Popup = () => {
+  // Button logic
+  const [buttonRecordActive, setButtonRecordActive] = useState(false);
+  const [buttonPlayActive, setButtonPlayActive] = useState(false);
+  const [recordText, setRecordText] = React.useState("Record");
```
Comment (agamm) [inline src/pages/Popup/Popup.jsx:12]: First, why both `React.useState` and `useState`?

File: src/pages/Popup/Popup.jsx [hunk 2]
```diff
@@ -1,62 +1,67 @@
...
+  function recordHandler() {
+    if (buttonRecordActive) {
+      setRecordText("Record");
+      setRecordIconClass("fa-solid fa-circle-dot");
+      setButtonRecordActive(false);
+    }
+    else {
+      if (buttonPlayActive) {
+        const element = document.querySelector('#playButton') || null;
+        if (element) element.click();
```
Comment (agamm) [inline src/pages/Popup/Popup.jsx:28]: Why are you clicking? this is very odd to do programmaticaly here.

**Review by agamm — CHANGES_REQUESTED** [review]
React related CR ;)

File: src/pages/Popup/Popup.jsx (hunk 1 above)
Comment (itoogii) [reply src/pages/Popup/Popup.jsx:12]: oh you are right. Let me correct it.

File: src/pages/Popup/Popup.jsx (hunk 2 above)
Comment (itoogii) [reply src/pages/Popup/Popup.jsx:28]: I wanted to make sure the other button is not active. Let me do it in other way.

File: src/pages/Popup/Popup.jsx [hunk 3]
```diff
@@ -1,62 +1,67 @@
...
+  // Button logic
+  const [buttonRecordActive, setButtonRecordActive] = useState(false);
+  const [buttonPlayActive, setButtonPlayActive] = useState(false);
+  const [recordText, setRecordText] = React.useState("Record");
+  const [recordIconClass, setRecordIconClass] = React.useState("fa-solid fa-circle-dot");
+  const [playText, setPlayText] = React.useState("Play");
+  const [playIconClass, setPlayIconClass] = React.useState("fa-solid fa-circle-play");
+  const buttonRecordClass = (buttonRecordActive) ? 'Button-stop' : 'Button-record';
+  const buttonPlayClass = (buttonPlayActive) ? 'Button-stop' : 'Button-play';
+
```
[the remark this answers is not in the document]
Comment (itoogii) [reply src/pages/Popup/Popup.jsx:18]: oh let me see what i can do. :)

**Comment by itoogii** [comment]
Updated the code and pushed for new request.

File: src/pages/Popup/Popup.jsx [hunk 4]
```diff
@@ -72,8 +63,10 @@ const Popup = () => {
         </form>
       </div>
       <div className="Button-area">
-        <button className="Button-style Button-record" id="recordButton" type="button"><i className="fa-solid fa-circle-dot"></i> Record</button>
-        <button className="Button-style Button-play" id="playButton" type="button"><i className="fa-solid fa-circle-play"></i> Play</button>
+        {buttonRecordActive || recordButton}
```
Comment (agamm) [inline src/pages/Popup/Popup.jsx:66]: You can rewrite to something like:
```
{{ buttonRecordActive? recordButton: stopRecordButton}
{{ buttonPlayActive ? playButton : stopPlayButton}
```

Right?

File: src/pages/Popup/Popup.jsx [hunk 5]
```diff
@@ -1,61 +1,52 @@
...
-        elRec.classList.add("Button-stop");
-        elRec.innerHTML = "<i class='fa-solid fa-circle-dot'></i> Stop";
-      }
-      else {
-        elRec.classList.remove("Button-stop");
-        elRec.classList.add("Button-record");
-        elRec.innerHTML = "<i class='fa-solid fa-circle-dot'></i> Record";
+
+  function recordHandler() {
+    if (buttonRecordActive) {
```
Comment (agamm) [inline src/pages/Popup/Popup.jsx:20]: You can rewrite all of this like:
function recordHandler() {
setButtonRecordActive((prev) => !prev);
}

Also in the play button.

**Review by agamm — CHANGES_REQUESTED** [review]
Much better, a few minor notes.

File: src/pages/Popup/Popup.jsx (hunk 4 above)
Comment (itoogii) [reply src/pages/Popup/Popup.jsx:66]: Oh I thought the other way is the way to do it in react :) 
You are right, let me update it.

File: src/pages/Popup/Popup.jsx (hunk 5 above)
Comment (itoogii) [reply src/pages/Popup/Popup.jsx]: I see. This makes code much neat. I have updated the code accordingly.

**Review by agamm — APPROVED** [review]
Looks great!

**Pull request merged**